test/WebJobs.Extensions.DurableTask.Analyzers.Test/ActivityFunction/FunctionReturnTypeAnalyzerTests.cs

```diff
@@ -69,6 +67,13 @@ public static string SayHello([ActivityTrigger] IDurableActivityContext context)
             return $""Hello {name}!"";
         }
 
+        [FunctionName(""E1_SayHello_Array"")]
+        public static string[] SayHello([ActivityTrigger] IDurableActivityContext context)
+        {
+            string name = context.GetInput<string>();
+            return $""Hello {name}!"";
```

Comment:
This example is a little strange because it's not valid C#. To avoid confusion, could we return `new [] { $@""Hello {name}!"" }` instead?

Reply:
This is valid C#, we use it in our sample "E1_SayHello"
`return $"Hello {name}!";`

I took most of this from our samples and just changed what I needed to to check the things I'm testing while making it resemble durable functions code.

Reply:
By not valid C#, I meant that it won't compile. This method signature defines the return value as `string[]` but you're returning a single `string`. The "E1_SayHello" sample has a `string` return type.

Reply:
These tests are all probably confusing because they are just a code chunk and you can't tell what you expect to be analyzed and diagnosed just by looking at it. Unlike usual unit tests which are usually easy to tell what is being tested by reading the code. I don't know how to get around that. But I'll fix these details as we notice them.

Reply:
One simple strategy might be to just add comments in the code chunks explaining what the expectation is.

Reply:
I'm not a big fan of adding a bunch of comments in my test files to explain exactly what my analyzer is doing because if my analyzer logic changes then the comments will be out of date. I still think the test classes are hard to read, but knowing the ArgumentAnalyzer test file contains test cases for the ArgumentAnalyzer is probably good enough.

Reply:
However, if you think that I need to change the names of my methods to clarify what my test cases are exactly, let me know.